lib/components/util/dropdown.tsx

```diff
@@ -161,7 +161,7 @@ const Dropdown = ({
       id={`${id}-wrapper`}
```

Comment:
I learned about `aria-live` while looking over the accessibility guidelines. Maybe adding an `aria-live="polite"` would help the screen reader get notified when the language changes languages in the dropdown:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Reply:
I'm not sure if this is what that is meant for? Although I might be confused. I see this more as being about an autocomplete suggestions dropdown rather than this which really could be considered more of a submenu? Not sure what others think

Reply:
I think the checkmark next to English is the selected indication right?

Reply:
Yeah, I just tried this on screenreader and it reads the selected language as "clickable, selected"!

Reply:
Tried it yesterday and for some reason i didn't catch the "selected" which is why I brought up the potential invisible labels. I think this is good the way it is!